plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaMetadata.java

```diff
@@ -247,6 +282,34 @@ public ConnectorTableProperties getTableProperties(ConnectorSession session, Con
             return Optional.empty();
         }
 
+        // FilterNode is not added, only if all domain match the following conditions:
```

Comment:
Can we handle them in a separate PR ?

Reply:
Do you mean to submit the applyfilter part separately？